### ISSUE TITLE
Implement Tron's version of uniswap

### ISSUE THREAD

**Issue by hackfisher** [evolutionlandorg/tron-contracts#59]
https://github.com/PhABC/uniswap-solidity

https://github.com/Uniswap/old-solidity-contracts/blob/master/contracts/Exchange/UniswapExchange.sol

Could be used as reference

**Comment by kevholder** [comment]
Hey just wanted to let you know that we (oikos.cash) have ported Uniswap to tron: https://swap.oikos.cash Github repo: https://github.com/oikos-cash/swap-solidity

**Comment by mirzausman371** [comment]
> Hey just wanted to let you know that we (oikos.cash) have ported Uniswap to tron: https://swap.oikos.cash Github repo: https://github.com/oikos-cash/swap-solidity

No, Your code is not working nor repo dev dependencies are available. Are you guys claiming for open source but not fully transparent?
I've been trying to access you but you didn't reply nor your Community leads dev. It would be really good if you still help me.. Thanks

**Comment by joshh71390** [comment]
likewise it's impossible to find anything that works and hasnt been gutted if it its on the tron blockchain... people hate helping eachother
